src/snovault/elasticsearch/__init__.py

```diff
@@ -16,12 +18,79 @@
 )
 import json
 import sys
+
+from redis import StrictRedis
+
+
 PY2 = sys.version_info.major == 2
 
 
 ROOT_LOG = logging.getLogger('root')
 
 
+class LocalStoreClient():
+    client = None
+    
+    def __init__(self, redis_db=0, local_store=None):
+        if local_store:
+            self.client = StrictRedis(
```

Comment:
we should have a fallback (try/catch) and default to the ephemeral storage objects if Redis fail. It will also prevent the class to throw exceptions up